src/control/cmd/dmg/system.go

```diff
@@ -210,20 +214,56 @@ func (cmd *systemStopCmd) Execute(_ []string) (errOut error) {
 	return resp.Errors()
 }
 
-type baseExcludeCmd struct {
-	baseCmd
-	cfgCmd
-	ctlInvokerCmd
-	cmdutil.JSONOutputCmd
-	rankListCmd
+// systemStartCmd is the struct representing the command to start system.
+type systemStartCmd struct {
+	baseRankListCmd
 }
 
-func (cmd *baseExcludeCmd) execute(clear bool) error {
+// Execute is run when systemStartCmd activates.
+func (cmd *systemStartCmd) Execute(_ []string) (errOut error) {
```

Comment:
minor - better not to shuffle the commands around if you can avoid it. Makes it look like more was changed than actually was.

Reply:
yes agreed but this has to be balanced against ensuring we have some ordering consistency across files which makes it easier to develop

Reply:
Agreed in general, although in this case it made the changes harder to review. In the diff, `systemStartCmd` moved (presumably didn't change?) and was treated as new code. The new command ended up where `systemStartCmd` used to be and the diff portrays it as if you edited that command. I don't need a revert or anything, since I've already gotten through it, but in future it would be best to make those kinds of code moves separately from big PRs like this.